autoload/ale/handlers/eslint.vim

```diff
@@ -44,7 +44,7 @@ function! ale#handlers#eslint#GetCommand(buffer) abort
 
     return ale#node#Executable(a:buffer, l:executable)
     \   . (!empty(l:options) ? ' ' . l:options : '')
-    \   . ' -f json --stdin --stdin-filename %s'
+    \   . ' -f json --stdin --stdin-filename %s 2> /dev/null'
```

Comment:
This will break the linter on Windows, and also some of the other parsing for detecting and reporting errors from ESLint.

I recommend finding some other way to fix this. Maybe the error parser can be updated with tests including example output which causes the problem.